internal/types/organization.go

```diff
@@ -31,7 +31,7 @@ type OrganizationWithBranding struct {
 }
 
 type OrganizationBranding struct {
-	ID                     uuid.UUID  `db:"id" json:"id"`
+	ID                     *uuid.UUID `db:"id" json:"id"`
```

Comment:
Is this change intentional? Because we never user pointers for the ID in any other DB models…

Reply:
No, recerted